packages/react-instantsearch/src/components/StarRating.js

```diff
@@ -161,7 +168,24 @@ class StarRating extends Component {
         })
       );
     }
-    return <div {...cx('root', !canRefine && 'noRefinement')}>{items}</div>;
+    return (
+      <BaseWidget
+        widgetClassName={widgetClassName}
+        otherWidgetClassNames={[!canRefine && `-noRefinement`]}
+        header={header}
+        footer={footer}
+      >
+        <svg xmlns="http://www.w3.org/2000/svg" style={{ display: 'none' }}>
```

Comment:
Maybe we can simply put this markup into a constant variable, then using it for pushing the stars in the array. Rather than use the `use` syntax. Don't have strong opinions about that, WDYT?

Reply:
Also possible to use https://github.com/smooth-code/svgr and have real svg images (need to build the rollup plugin)

Reply:
I'd rather stick to `use` as it is the recommended way to deal with repeating SVGs.